src/Core/Content/Mail/Service/MailSender.php

```diff
@@ -71,6 +72,16 @@ public function send(Email $email, ?Envelope $envelope = null): void
         }
 
         $mailData = serialize($email);
+        if (\strlen($mailData) <= self::MESSAGE_SIZE_LIMIT) {
```

Comment:
should we merge the `mailer->send` calls? 
```suggestion
        if (\strlen($mailData) <= self::MESSAGE_SIZE_LIMIT || $this->messageBus === null) {
```

Reply:
Originally had the same idea, but that would always call `serialize` and `strlen` even though we might not need to call them.

Reply:
nice, I missed that